### PR TITLE
Vis.js 4.21.0

### PR DESCRIPTION
**Pull request by greg-minshall**
adding the odd "T" for vis.js 4.21.0 compatability.  fix some documentation to follow this, and also a code example (where the documentation of the data structure had deviated from the actual code for that data structure).

### CODE REVIEW

**Comment by daattali** [comment]
This PR assumes that timevis 4.21 is already being used, right? If I just merge it right now, it'll break because timevis doesn't use it

File: inst/example/ui-helpers.R [hunk 1]
```diff
@@ -14,7 +14,7 @@ data <- data.frame(
   content = c("Item one", "Item two",
               "Ranged item", "Item four"),
   start   = c("2016-01-10", "2016-01-11",
-              "2016-01-20", "2016-02-14 15:00:00"),
+              "2016-01-20", "2016-02-14"),
```
Comment (daattali) [inline inst/example/ui-helpers.R]: is there a specific reason the time of day here was removed?

Comment (greg-minshall) [reply inst/example/ui-helpers.R]: a) i think this also works with 4.16.1.

b) the code in the example didn't match with the code in the example.  err, i.e., what was displayed to the user as the code included the time of day, but the code that generated the timeline displayed to the user did *not* include the time of day.  (modulo my being brain-dead.)

**Comment by daattali** [comment]
@greg-minshall thanks for this PR - there are still too many regression bugs with the new vis.js library so I'm still holding off on upgrading, but when the time comes, I will use this information, it was useful that you pointed out how to make 4.21 work. Thanks!

**Pull request closed without merging**